src/vcpkg/commands.upgrade.cpp

```diff
@@ -24,11 +24,13 @@ namespace vcpkg::Commands::Upgrade
 
     static constexpr StringLiteral OPTION_NO_DRY_RUN = "no-dry-run";
     static constexpr StringLiteral OPTION_KEEP_GOING = "keep-going";
+    static constexpr StringLiteral OPTION_STOP_AT_FAILURE = "stop-at-failure";
```

Comment:
```suggestion
    static constexpr StringLiteral OPTION_NO_KEEP_GOING = "no-keep-going";
```

this is the standard.